packages/mobile/src/screens/notifications-screen/NotificationsScreen.tsx

```diff
@@ -17,6 +18,7 @@ const messages = {
 }
 
 export const NotificationsScreen = () => {
+  usePopToTopOnDrawerOpen()
```

Comment:
Do you need this? Because this only makes sense for screens to will be navigated to from the notification drawer

Reply:
so it has the navigation setting in there as well, so that's main reason why. but once we remove the old way, we can remove it and just have the stack nav setting (assuming we still need that, even)

Reply:
Yeah makes sense, easiest just to leave it!